app/helpers/application_helper.rb

```diff
@@ -1,2 +1,7 @@
 module ApplicationHelper
+  def options_for_video_reviews(selected = nil)
+    options = [5,4,3,2,1].map { |number| [pluralize(number, "Star")]}
```

Comment:
Good to know, doing it this way makes it so I don't have to pluralize the selected value.